### PR TITLE
Delete product method created

### PR DESCRIPTION
**Pull request by omersolmazer**
Delete product method is available for review

### CODE REVIEW

**Review by BobZcao — APPROVED** [review]
Thank you for implementing the DELETE. Actually I am doing that part.

File: server.py [hunk 1]
```diff
@@ -41,3 +41,26 @@ def update_product(id):
 
   """
   pass
+
+@app.route('inventory/products/<int:id>', methods=['DELETE'])
```
Comment (p2hang) [inline server.py]: Looks good!
Just one thing I'm not sure about, should we have the '/' before the `inventory/products/`?
like `/inventory/products/<int:id>`

**Review by p2hang — CHANGES_REQUESTED** [review]
Looks good, just one tiny problem might need fix.

**Review by pinocchiochang — APPROVED** [review]
OK. Approved.

**Review by pinocchiochang — APPROVED** [review]
Good to me.

**Review by p2hang — APPROVED** [review]
Looks good!

**Comment by omersolmazer** [comment]
Code updated with reviews

**Pull request merged**